modules/installation-gcp-permissions.adoc

```diff
@@ -29,6 +29,267 @@ account requires the following permissions. If you deploy your cluster into an e
 * Service Account User
 * Storage Admin
 
+The following are the required permissions for provisioning GCP infrastructure for creating and deleting the {product-title} cluster.
+
+.Required permissions for creating network resources
+[%collapsible]
+====
+* `compute.addresses.create`
+* `compute.addresses.createInternal`
+* `compute.addresses.get`
+* `compute.addresses.delete`
+* `compute.addresses.list`
+* `compute.addresses.use`
+* `compute.addresses.useInternal`
+* `compute.firewalls.create`
+* `compute.firewalls.get`
+* `compute.firewalls.delete`
+* `compute.firewalls.list`
```

Comment:
Suggest to rearrange the permissions in ascending order as below
* `compute.firewalls.create`
* `compute.firewalls.delete`
* `compute.firewalls.get`
* `compute.firewalls.list`

Reply:
Addressed